docs/manual/layout/templates/manage-template.de.md

```diff
@@ -3,8 +3,7 @@ title: "Templates verwalten"
 description: "Die Verwaltung der Template-Dateien."
 url: "layout/templates/verwaltung"
 aliases:
-    - /de/templates/verwaltung/
```

Comment:
This deletes an alias. Was the article never linked on that alias?

Reply:
No, there wass only one file within /de/theme-manager/templates-verwalten/ before.